project/src/pages/room/components/gallery.tsx

```diff
@@ -2,23 +2,21 @@ import { useSelector } from 'react-redux';
 import { InitialState } from '@customTypes/store';
 
 function Gallery(): JSX.Element {
-  const imagesCollection = useSelector((state: InitialState) => state.offerToShow.images);
+  const imagesCollection = useSelector((state: InitialState) => state.offerToShow?.images);
 
   return (
     <div className="property__gallery-container container">
       <div className="property__gallery">
-        {
-          imagesCollection.map((images) =>
-            (
-              <div key={images} className="property__image-wrapper">
-                <img
-                  className="property__image"
-                  src={images}
-                  alt="studio"
-                />
-              </div>)
-          )
-        }
+        { imagesCollection?.slice(0).sort(() => Math.random() - 0.5).slice(0, 6).map((images) =>
```

Comment:
```suggestion
        {
          imagesCollection?.slice(0).sort(() => Math.random() - 0.5).slice(0, 6).map((images) =>
```

вообще, не понял, а зачем тебе тут слайс, потом да еще в псевдо рандоме, а потом еще один слайс?

Reply:
++